code/modules/power/lights.dm

```diff
@@ -263,12 +263,12 @@
 	switch(fitting)
 		if("tube")
 			brightness_range = 8
-			if(prob(2))
+			if(prob(50))
```

Comment:
Why not prob(99)?
it's balanced i swear

Reply:
you're right, this is too low. I'll add more randomness to buff it

Reply:
👍